src/sayt/sayt.ts

```diff
@@ -0,0 +1,20 @@
+/** The name of event fired when SAYT should be hidden. */
+export const SAYT_HIDE = 'sfx::sayt_hide';
+/** The type of the [[SAYT_HIDE]] event payload. */
+export type SaytHidePayload = SaytVisibilityPayload;
+
+/** The name of event fired when SAYT should be hidden. */
```

Comment:
Same as my comment from line 1 - either remove 'the' in autocomplete doc notes, or add in sayt's?

Reply:
Updated.